internal/version/version.go

```diff
@@ -16,6 +16,3 @@ package version
 
 // Version variable will be replaced at link time after `make` has been run.
 var Version = "latest"
-
-// GitHash variable will be replaced at link time after `make` has been run.
-var GitHash = "<NOT PROPERLY GENERATED>"
```

Comment:
Should this be removed here?  Even if it isn't used I'd probably prefer to remove it in a separate PR to maintain the focus of this PR on `ApplicationStartInfo` -> `BinaryInfo`.

Reply:
I will add this back